.github/dependabot.yml

```
@@ -0,0 +1,32 @@
+---
+
+# Any ignore directives should be uncommented in downstream projects to disable
+# Dependabot updates for the given dependency. Downstream projects will get
+# these updates when the pull request(s) in the appropriate skeleton are merged
+# and Lineage processes these changes.
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    ignore:
+      - dependency-name: actions/cache
+      - dependency-name: actions/checkout
+      - dependency-name: actions/setup-python
+
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    ignore:
+      - dependency-name: "ansible"
+      - dependency-name: "ansible-lint"
+
+  - package-ecosystem: "terraform"
+    directory: "/terraform"
+    schedule:
+      interval: "weekly"
+    # ignore:
+    #   - dependency-name: "hashicorp/aws"
```

We want to uncomment this bit inherited from `skeleton-ansible-role-with-test-user`:
```suggestion
    ignore:
      - dependency-name: "hashicorp/aws"
```